content/intro/pricing/whats-included.md

```diff
@@ -0,0 +1,35 @@
+---
+menu:
+  main:
+    parent: pricing
+title: What's included
+weight: 10
+---
+
+All cloud.gov access packages include the following: 
```

Comment:
It might be good to get some plain language here. I'm thinking about someone on the $$$ side looking to understand the below.